### ISSUE TITLE
The weight and the bias in the Batch Norm

### ISSUE THREAD

**Issue by UpCoder** [pytorch/pytorch#16149]
Hi, I am not sure the meaning of the weight and the bias in the Batch Normalization. Does it represent the gamma and the beta?
Thank you very much!

**Comment by elvisyjlin** [comment]
Yes, they are. The weight and bias in [`_BatchNorm`](https://pytorch.org/docs/stable/_modules/torch/nn/modules/batchnorm.html) are the gamma and beta in the documentation of [`torch.nn.BatchNorm1d`](https://pytorch.org/docs/stable/nn.html#batchnorm1d), [`torch.nn.BatchNorm2d`](https://pytorch.org/docs/stable/nn.html#batchnorm2d), and [`torch.nn.BatchNorm3d`](https://pytorch.org/docs/stable/nn.html#batchnorm3d). I think the reason they're named as weight and bias is that they perform an affine transformation (w*x+b) to normalized inputs and they are trainable tensors.

For detailed computation of Batch Normalization, please refer to the source code [here](https://github.com/pytorch/pytorch/blob/83c054de481d4f65a8a73a903edd6beaac18e8bc/torch/csrc/jit/passes/graph_fuser.cpp#L232). The input is normalized by the calculated mean and variance first. Then the transformation of w*x+b is applied on it by adding the operations to the computational graph.

**Comment by UpCoder** [comment]
@elvisyjlin OK, your explanation is very clear. Thank you very much!
